index.js

```diff
@@ -80,6 +80,16 @@ app.post('/start', (req, res) => {
   if (req.body.passphrase) {
     walletConfig['passphrase'] = req.body.passphrase;
   }
+  const walletID = req.body['wallet-id'];
+
+  if (walletID in wallets) {
+    // We already have a wallet for this key
+    // so we stop the old wallet and start the new one
+    const oldWallet = wallets[walletID];
+    oldWallet.stop();
+
+    delete wallets[walletID];
+  }
```

Comment:
Why do it do it like this? Can't we just return saying this wallet is already started?

Reply:
Ah, didn't think about this other use. Actually, I think we should then return an error if he tries to start a wallet with an id already in use. He should then explicitly stop it and start another one.

Reply:
Yeah, I agree with you. Changed the code to have this behaviour.